nutkit/protocol/responses.py

```diff
@@ -226,7 +226,7 @@ def __eq__(self, _):
                     "Backend supports well-formatted counter. "
                     "Remove the backwards compatibility check!"
                 )
-        if get_driver_name() in ["java", "javascript", "go", "dotnet"]:
+        if get_driver_name() in ["java", "javascript", "go", "dotnet", "ruby"]:
```

Comment:
Same here: If you driver has the option to provide this info, It'd be the preferred way to send the data. If not, this is fine.

It should be a rather small change in your back end implementation should the driver already offer the data.